backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -903,6 +904,10 @@ databaseChangeLog:
                 ordering_provider_id,
                 default_device_type
               FROM ${database.defaultSchemaName}.organization;
+      rollback:
+        - sql:
+            sql: |
+              TRUNCATE TABLE ${database.defaultSchemaName}.facility;
```

Comment:
Does Liquibase let you edit previous migrations like this? My understanding was that would violate the checksums.

Reply:
Turns out that it does! Only editing the `changes` changes the checksum - the rollback is apparently not checksummed.

Reply:
Perfect! That makes our lives much easier 😄